src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -280,13 +281,24 @@ class DistributedClosestPointImpl
   DistributedClosestPointImpl(RuntimePolicy runtimePolicy, bool isVerbose)
     : m_runtimePolicy(runtimePolicy)
     , m_isVerbose(isVerbose)
+    , m_sqDistanceThreshold(std::numeric_limits<double>::max())
   {
     setAllocatorID();
 
     MPI_Comm_rank(MPI_COMM_WORLD, &m_rank);
     MPI_Comm_size(MPI_COMM_WORLD, &m_nranks);
   }
 
+  /**
+   * \brief Sets the threshold for the query
+   *
+   * \param [in] threshold Ignore distances greater than this value.
+   */
+  void setSquaredDistanceThreshold(double sqThreshold)
+  {
+    m_sqDistanceThreshold = sqThreshold;
```

Comment:
Consider a check for non-negative value (such as at (new) line 957 in `setDistanceThreshold`.

Reply:
I considered it and originally decided that the check would be redundant because we have a check in the public interface already.  Do it anyway?

Reply:
Well, `setSquaredDistanceThreshold()` is a `public` member function and does have Doxygen, so I'd recommend checking here.  Some enterprising user will be sure to find it and try calling it, doubtless with values that "for sure" won't be negative : )

Reply:
Okay.  Will do.  Thanks.

Reply:
I pushed the check.